src/components/Reports/ReportsFilters/ReportsFilters.js

```diff
@@ -39,7 +39,9 @@ const ReportsFilters = ({
                 {domainSelected.verified_date ? (
                   <>
                     <FormattedMessage id="reports_filters.verified_domain" />{' '}
-                    <FormattedDate value={domainSelected.verified_date} timeZone="UTC" />
+                    <span>
```

Comment:
Out of curiosity, why do you need an extra span here?

Reply:
in previous versions the `span` is generated automatically now dont

Reply:
And I asume you need it here because of styles or anything else, ok. Thanks!